backtester/eventhandlers/exchange/exchange.go

```diff
@@ -103,18 +104,20 @@ func (e *Exchange) ExecuteOrder(o order.Event, data data.Handler, bot *engine.En
 	} else {
 		limitReducedAmount = reducedAmount
 	}
+
+	log.Infof(log.BackTester, "limitReducedAmount %v, direction %v,  buyside %+v, sell size %+v", limitReducedAmount, f.GetDirection(), cs.BuySide, cs.SellSide)
 	// Conforms the amount to fall into the minimum size and maximum size limit after reduced
 	switch f.GetDirection() {
 	case gctorder.Buy:
-		if (limitReducedAmount < cs.BuySide.MinimumSize || limitReducedAmount > cs.BuySide.MaximumSize) && (cs.BuySide.MinimumSize > 0 && cs.BuySide.MaximumSize > 0) {
+		if (limitReducedAmount < cs.BuySide.MinimumSize || limitReducedAmount > cs.BuySide.MaximumSize) && cs.BuySide.MaximumSize > 0 {
```

Comment:
This change still means that if a user has a minimum size set, but not a maximum, then it won't be hit.
eg I have `cs.BuySide.MinimumSize = 1`, the `limitReducedAmount = 0.5`, but I have `cs.BuySide.MaximumSize = 0`, then an order is allowed to be placed at 0.5.
`if limitReducedAmount < cs.BuySide.MinimumSize || (limitReducedAmount > cs.BuySide.MaximumSize && cs.BuySide.MaximumSize > 0)`

Reply:
@gloriousCode updated